capella_rm_bridge/__main__.py

```diff
@@ -189,7 +189,8 @@ def main(
             ERROR_PATH.write_text(error_statement, encoding="utf8")
             LOGGER.info("Change-errors file %s written.", ERROR_PATH)
 
-        sys.exit(1)
+        if not force:
+            sys.exit(1)
```

Comment:
`--force` says:

> If a non RequirementModule-error was encountered only the object and all related objects will be skipped. Intact objects will still be synchronized.

I would understand this as "ignore *this one specific type* of error", but this change will ignore *every* error. Which one's supposed to be correct here? If it's supposed to be "ignore every error", could you please update the help text to make that a bit clearer?